deploy/06_deploy_coverage_pool.js

```diff
@@ -1,23 +1,35 @@
 module.exports = async ({ getNamedAccounts, deployments }) => {
-  const { deploy } = deployments
+  const { deploy, read, execute } = deployments
   const { deployer } = await getNamedAccounts()
   const AssetPool = await deployments.get("AssetPool")
-  const UnderwriterToken = await deployments.get("UnderwriterToken")
 
   const CoveragePool = await deploy("CoveragePool", {
     from: deployer,
     args: [AssetPool.address],
     log: true,
   })
 
-  const assetPool = await ethers.getContractAt("AssetPool", AssetPool.address)
-  const underwriterToken = await ethers.getContractAt(
-    "UnderwriterToken",
-    UnderwriterToken.address
-  )
+  if ((await read("AssetPool", "owner")) !== CoveragePool.address) {
```

Comment:
Shouldn't we check addresses in a more secure way? I mean we should compare checksum addresses, unless we are sure that addresses are in the same "format". https://docs.ethers.io/v5/api/utils/address/#utils-getAddress.

```
ethers.utils.getAddress((await read("AssetPool", "owner")) !== ethers.utils.getAddress(CoveragePool.address)
```

Reply:
Great point! I will handle it in a follow-up PR.

Reply:
https://github.com/keep-network/coverage-pools/pull/153